src/components/workflow/WorkflowList.vue

```diff
@@ -184,6 +213,19 @@ export default {
       sharedService: new SharedService(),
       rightArrowSvg: config.common.icons["right_arrow"],
       activeWorkflowSession: "",
+      userGuideUrl: env.getUserGuideUrl(),
+      workflowToolList: {
+        extract_audio: "Extract Audio",
+        applause_detection: "Applause Detection",
+        aws_transcribe: "AWS Transcribe",
+        aws_comprehend: "AWS Comprehend",
+        tesseract_video_ocr: "Tesseract Video OCR",
+        azure_video_indexer: "Azure Video Indexer",
+        pyscenedetect: "PySceneDetect",
+        dlib_face_recoginition: "Dlib Face Recoginition",
+        azure_video_ocr: "Azure Video OCR",
+        contact_sheets: "Contact Sheets",
+      },
     };
```

Comment:
why do we need this mapping? the MGM id and name are defined by MGM config. and both are available in the API response. I think what you need is the mapping between env property name and the MGM id, as you have defined in the router.js.

Can you change workflowToolList to mgmList?

Reply:
![Screenshot from 2022-06-11 01-20-42](https://user-images.githubusercontent.com/102983633/173372067-521d0c11-94d7-4088-982f-621ea33fc6b5.png)

I've created this object to filter out the the key of mgmList when user is clicking on "Tool documentation" button.
Once I got the key same key I'm searching in router.js and passing the router path.

Reply:
I didn't find any other way to know, user clicked on which mgmList item(nav bar)'s "Tool documentation" button.